Point_set_processing_3/examples/Point_set_processing_3/CMakeLists.txt

```diff
@@ -25,13 +22,9 @@ endif()
 # Activate concurrency?
 option(CGAL_ACTIVATE_CONCURRENT_PSP3 "Enable concurrency" OFF)
 
-set(CGAL_libs CGAL::CGAL)
 if(CGAL_ACTIVATE_CONCURRENT_PSP3 OR "$ENV{CGAL_ACTIVATE_CONCURRENT_PSP3}")
   find_package(TBB REQUIRED)
   include(CGAL_TBB_support)
-  if(TARGET CGAL::TBB_support)
-    set(CGAL_TBB_target ${CGAL_libs} CGAL::TBB_support)
-  endif()
 endif()
 
```

Comment:
With `CGAL_ACTIVATE_CONCURRENT_PSP3`, TBB is searched, but never used.

The problems date from PR #4636. See the culprit change at https://github.com/CGAL/cgal/pull/4636/files#diff-16a24da0dde4987fe4dfe346bf9547c0985d8e02341d32fbe191b6197d0ea1f8.

Reply:
Shall we fix this in this PR, or shall we handle that independently so that it also makes it into 5.6.1 ?

Reply:
I think it is better in an other PR. This one is already complex enough.

Reply:
@soesau can you please take care of this.

Reply:
I have created https://github.com/CGAL/cgal/issues/8553 to track that issue.